### DISCUSSION TITLE
Failed with exit code 12

### DISCUSSION THREAD

**Discussion by atistabiur** [frePPLe/frepple#532] [Q&A]
When running step 6 with the demo data i got this error:
"Failed with exit code 127"
with the arguments:
"--constraint=13 --plantype=1 --env=supply"

Can you help on addressing this issue ?"

**Comment by jdetaeye** [comment]
What's at the end of the log file?  

If there is no log file, then it could indicate an incomplete or incorrect installation.  How did you install the software? Which version?

**Reply by MaxbyteTechnologies** [reply]
Same problem here. How to resolve this?

**Reply by jdetaeye** [reply]
With information on error message, context on dataset being used, version, which platform, ... some more rational diagnosis would be possible.   

However, with the level of information being shared on this thread, this can only be resolved with magic or divine inspiration. :-)

**Reply by MaxbyteTechnologies** [reply]
Followed the installation instruction for ubuntu "DEBIAN INSTALLATION SCRIPT". After installation I was able to open the localhost and create the items, operations... But while trying to create manufacturing order and execute, the above error was shown.

"Failed with exit code 127"
with the arguments:
"--constraint=13 --plantype=1 --env=supply"

Just restarted the server post which the frepple web page is not opening. It is showing "500 Internal Server Error"

**Reply by MaxbyteTechnologies** [reply]
Pls help in understanding the troubleshooting steps for checking if frepple is properly installed or not.

**Reply by mike-ivc-inform** [reply]
Hello!
The first step is: - Tell us what version of the operating system you have and what version of the program.
Second step: take a screenshot with program messages (command line, terminal). After that, other participants will have an understanding of what needs to be done next and, perhaps, the participants will suggest a solution to you

**Reply by MaxbyteTechnologies** [reply]
Ubuntu 22.04.1 LTS
![Screenshot from 2022-09-01 13-55-26](https://user-images.githubusercontent.com/53274626/187868420-902df0db-427e-490e-9c04-e3e22d3a1a0f.png)

**Reply by mike-ivc-inform** [reply]
how do you run the program?

**Reply by jdetaeye** [reply]
Ubuntu 22 is not supported.   We only provide installers for ubuntu 18 & 20.   Using installers from different OS versions is looking for trouble.

Check the web server log file /var/log/apache2/error.log for more details on what's wrong.

**Reply by MaxbyteTechnologies** [reply]
I could not find any specific error in the log file. Attached here for reference.
[access.log](https://github.com/frePPLe/frepple/files/9468511/access.log)

**Reply by MaxbyteTechnologies** [reply]
It is working now after installing in Ubuntu 20.04. Thanks for the support.
